workers/Dockerfile

```diff
@@ -12,9 +12,9 @@ RUN pip3 install common/$(ls common -1 | sort --version-sort | tail -1)
 
 USER user
 # Install Aspera
-ADD --chown=user:user https://download.asperasoft.com/download/sw/cli/3.7.7/aspera-cli-3.7.7.608.927cce8-linux-64-release.sh aspera-cli-3.7.7.608.927cce8-linux-64-release.sh
+RUN wget -q https://download.asperasoft.com/download/sw/cli/3.7.7/aspera-cli-3.7.7.608.927cce8-linux-64-release.sh
+ADD --chown=user:user aspera-cli-3.7.7.608.927cce8-linux-64-release.sh aspera-cli-3.7.7.608.927cce8-linux-64-release.sh
 RUN sh aspera-cli-3.7.7.608.927cce8-linux-64-release.sh
-RUN rm aspera-cli-3.7.7.608.927cce8-linux-64-release.sh
```

Comment:
Why did you remove this?

Reply:
So Docker doesn't re-download the file every time the script is ran.